fineract-core/src/main/java/org/apache/fineract/organisation/monetary/domain/MoneyHelper.java

```diff
@@ -59,4 +63,11 @@ public static MathContext getMathContext() {
         }
         return mathContext;
     }
+
+    @Profile(FineractProfiles.TEST)
```

Comment:
I dont think profile do anything here..l its a static method...part of the class... :/
Also MoneyHelper has not much to do with configuration rounding mode. 

Would you mind moving this logic into the ConfigurationDomainService without the profile annotation? The profile annotation do absolutely nothing here...

Reply:
Money helper init the rounding variable the first time with X value, so If you change the GC with the API or the database directly, this new value is not used until you restart Fineract

Reply:
Ok, but still the annotation can be removed, it is meaningless here

Reply:
also please either change the method name, as it is not reset the rounding mode, rather fetch it via configuraiton service again, or change the implementation to set the roundingMode to NULL

Reply:
Done! Annotation and method renamed